### PR TITLE
Added homework 15

### PR DESCRIPTION
**Pull request by Miadzvedz**
**Projects**
- HW15.Task1
- HW15.Task2

### CODE REVIEW

File: CSharp.Homeworks/hw_15/HW15.Task1/StringExtension.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        {
+            return text switch
+            {
+                "TRUE" => 1,
+                "FALSE" => 0,
+                _ => throw new ArgumentException("Argument is not TRUE or FALSE"),
+            };
+        }
+    }
+}
```
Comment (VadzimPapko) [inline CSharp.Homeworks/hw_15/HW15.Task1/StringExtension.cs:23]: good

File: CSharp.Homeworks/hw_15/HW15.Task2/CreditCardAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            Regex regN = new(@"\d{4}([\s-]\d{4}){3}");
+            Regex regL = new(@"\d{16}");
+            if (regN.Match(numCard).Success || regL.Match(numCard).Success)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
```
Comment (VadzimPapko) [inline CSharp.Homeworks/hw_15/HW15.Task2/CreditCardAttribute.cs:31]: you can use this expression instead
`return regN.Match(numCard).Success || regL.Match(numCard).Success`

File: CSharp.Homeworks/hw_15/HW15.Task2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            {
+                Console.WriteLine("Card is valid");
+            }
+            else
+            {
+                Console.WriteLine("Card is not valid");
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline CSharp.Homeworks/hw_15/HW15.Task2/Program.cs:27]: good

**Review by VadzimPapko — APPROVED** [review]
Reviewed. Your score is 9

File: CSharp.Homeworks/hw_15/HW15.Task2/CreditCardAttribute.cs (hunk 2 above)
Comment (Miadzvedz) [reply CSharp.Homeworks/hw_15/HW15.Task2/CreditCardAttribute.cs:31]: Of course, thank you :)

**Pull request merged**